README.md

```diff
@@ -88,3 +88,163 @@ In order to configure connection to elasticsearch you have to provide following
 * `ELASTICSEARCH_PASSWORD` - password for the user (see
   [official guide](https://www.elastic.co/guide/en/elasticsearch/reference/current/configuring-security.html)
   for more details about ES security).
+
+## Indexing
+
+### Recreating Elasticsearch index
+
+Sometimes we need to recreate Elasticsearch index, for example when a breaking change introduced to ES index
+structure (mapping).
+
+Here are the steps how to do it:
+1. Remove ES index:
+```http
+   DELETE [ES_HOST]/instance_[tenant_name]
+```
```

Comment:
Wondering if we should add mod-search API to abstract this?

Reply:
Make sense. I'm thinking about introducing a `recreateIndex=true` parameter for reindex API. I'll create a story for it.

Reply:
Here is the story - https://issues.folio.org/browse/MSEARCH-121